### PR TITLE
Module-1: Portfolio project Mobile version

### PR DESCRIPTION
**Pull request by Profsain**
## Module-1: Portfolio project Mobile version
- Added Portfolio recent works section
- Added about me section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -79,6 +83,155 @@ <h2 class="headerTitle greenText">I’m a software developer</h2>
...
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <button class="sectionBtn">Get My Resume</button>
+        <div class="sectionGrid">
+          <div class="sectionCard aboutCard">
+            <div class="iconHolder">
+              <img src="asset/image//icons/langauge.png" alt="langauge icons" />
+            </div>
+            <h3 class="sectionCardTitle">Languages</h3>
```
Comment (Danieldotcomcoder) [inline index.html:208]: Kindly add the technology categories ("Languages", "Frameworks" etc ...) in a `ul` tag each of them in a `li` tag , that will make your code more professional and also its better to use `ul` and `li` when listing elements✔😃

File: index.html [hunk 2]
```diff
@@ -79,6 +83,155 @@ <h2 class="headerTitle greenText">I’m a software developer</h2>
...
+            <div class="cardTags">
+              <p class="tagText aboutTags">JavaScript</p>
+              <p class="tagText aboutTags">Ruby</p>
+              <p class="tagText aboutTags">HTML</p>
+              <p class="tagText aboutTags">CSS</p>
```
Comment (Danieldotcomcoder) [inline index.html]: Kindly also add the technologies in each category in a `ul` tag each of them in a `li` tag that will make your code more professional ✔😃

PS: don't forget to do the same inside frameworks and skills too ✔😃

File: index.html [hunk 3]
```diff
@@ -79,6 +83,155 @@ <h2 class="headerTitle greenText">I’m a software developer</h2>
...
+              src="asset/image/icons/imgPlaceholder.png"
+              alt="project image placeholder"
+            />
+            <h3 class="sectionCardTitle">Multi-Post Stories Gain+Glory</h3>
+            <div class="cardTags">
+              <p class="tagText">Ruby on rails</p>
+              <p class="tagText">CSS</p>
+              <p class="tagText">JavaScript</p>
+              <p class="tagText">HTML</p>
+            </div>
```
Comment (Danieldotcomcoder) [inline index.html]: Kindly for each project in your projects section put the technologies (HTML, CSS, etc... ) inside a `ul` tag each of them inside a `li` tag to make your code look professional ✔😃👍

File: index.html [hunk 4]
```diff
@@ -79,6 +83,155 @@ <h2 class="headerTitle greenText">I’m a software developer</h2>
...
+            <button class="sectionBtn">See Project</button>
```
Comment (Danieldotcomcoder) [inline index.html:106]: For the 'see project' buttons kindly include the button interactions (hover, active, etc... ) from the Figma template ✔😃✔

image

**Review by Danieldotcomcoder — CHANGES_REQUESTED** [review]
Hello @Profsain, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters are setup correctly ✔✔
- Clear and nice code ✔✔
- All Linters are passing ✔✔

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html [hunk 5]
```diff
@@ -79,6 +83,173 @@ <h2 class="headerTitle greenText">I’m a software developer</h2>
...
+              <ul class="cardTags">
+                <li class="tagText aboutTags">Codekit</li>
+                <li class="tagText aboutTags">GitHub</li>
+                <li class="tagText aboutTags">Codepen</li>
+                <li class="tagText aboutTags">Gitlab</li>
+                <li class="tagText aboutTags">Terminal</li>
+              </ul>
+            </div>
+          </div>
+        </div>
```
Comment (medaminedev66) [inline index.html]: - [ ] As requested by the previous reviewer, the technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul` instead `div`, each of them in a `li`. Check out the code below:
```html
 <ul class="sectionGrid">
          <li class="sectionCard aboutCard">
            <div class="iconHolder">
              <img src="asset/image//icons/langauge.png" alt="langauge icons" />
            </div>
            <h3 class="sectionCardTitle">Languages</h3>
            ....
          </li>
  </ul>
```

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @Profsain 😃,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Your design looks great and matches the Figma design. Great job 👍 .

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 5 above)
Comment (Profsain) [reply index.html]: Thank you.

**Review by gtekle — APPROVED** [review]
Hi @Profsain :wave:, 

## Approved :clap: :clap: :clap: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved-stamp


## Highlights
:heavy_check_mark: Good use of Github flow
:heavy_check_mark: Nice UI
:heavy_check_mark: No linter errors

## Optional suggestions

[OPTIONAL] The background image for the headline is overflowing to the toolbar section thereby affecting the visibility of the hamburger menu button. Please take a look at the following snapshots for your implementation and Figma design and update it accordingly before merging this pull request.

#### Figma Design
headline-background-image-figma

#### Implementation
headline-background-image


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**